test/e2e-tests.sh

```diff
@@ -22,17 +22,21 @@ failed=0
 (( !failed )) && logger.success '🚀 Cluster prepared for testing.'
 
 # Run serverless-operator specific tests.
-(( !failed )) && run_e2e_tests || failed=4
+(( !failed )) && serverless_operator_e2e_tests || failed=4
 
 # Run upstream knative serving & eventing operator tests
 (( !failed )) && deploy_serverless_operator_latest || failed=11
-(( !failed )) && run_knative_serving_operator_tests || failed=12
-(( !failed )) && run_knative_eventing_operator_tests || failed=14
+(( !failed )) && knative_serving_operator_tests || failed=12
+(( !failed )) && knative_eventing_operator_tests || failed=14
 
 # Run upstream knative serving & eventing tests
 (( !failed )) && ensure_serverless_installed || failed=15
-(( !failed )) && run_knative_serving_e2e_and_conformance_tests || failed=16
-(( !failed )) && run_knative_eventing_tests || failed=17
+
+# Run knative serving additional e2e tests
+(( !failed )) && downstream_serving_e2e_tests || failed=5
+
+(( !failed )) && upstream_knative_serving_e2e_and_conformance_tests || failed=16
+(( !failed )) && knative_eventing_tests || failed=17
```

Comment:
This function is not renamed, but it is still `run_knative_eventing_tests`. Due to this all e2e with this PR is failing.

```
./test/e2e-tests.sh: line 39: knative_eventing_tests: command not found
```

Reply:
Fixed.